### ISSUE TITLE
Fatal error: Problematic frame: # C  [jna8893831892584865540.tmp+0x9220]

### ISSUE THREAD

**Issue by oncecrossover** [brevilo/jolm#9]
```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f8a5f99c220, pid=11858, tid=0x00007f8a624f4700
#
# JRE version: OpenJDK Runtime Environment (8.0_292-b10) (build 1.8.0_292-8u292-b10-0ubuntu1~16.04.1-b10)
# Java VM: OpenJDK 64-Bit Server VM (25.292-b10 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [jna8893831892584865540.tmp+0x9220]
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
...
```

Hey @brevilo, we deployed our service to the Linux product environment, The terrible thing is that our service has been down two times in one day, I think it is due to the jOlm framework. this may be the same problem as #4. 

I recommend that you deploy an API and then simulate concurrent testing, you will reproduce this error.

Could you quickly find the problem and fix it? that's important for us. Thanks!

**Comment by brevilo** [comment]
Well, I'm trying to help you here but you really need to do your part. It doesn't help to keep opening already suspected duplicate issues, to [keep not providing basic details](https://github.com/brevilo/jolm/issues/7#issuecomment-968337563) or to [keep forgetting](https://github.com/brevilo/jolm/issues/4#issuecomment-968308174) about the earlier issues submitted, presumably a consequence of opening multiple ones on the same topic.

That said, there's a (very likely) [solution waiting](https://github.com/brevilo/jolm/issues/4#issuecomment-970387652) for you to test - for more than 4 days now. "I recommend that you" go ahead and test it and report back in #4.

Closing this as a duplicate of #4
